### PR TITLE
Be consistent about our definition of fetch links

### PR DESCRIPTION
**Pull request by alexwlchan**
## Who is this for?

Devs.

## What is it doing for them?

Making it clear what fetchLinks are when you see a variable for them in the codebase.

Currently when you see a variable like `eventsFetchLinks`, it could mean one of two things:

1. The fields on non-event types that are linked on events, which we want to retrieve in `fetchEvents()`. e.g. if we want to get information on articles which are linked to events:

    ```typescript
    const eventsFetchLinks = ['article.title', 'article.promo'];
    ```

2. The fields on event types, which we want to retrieve when fetching linked events on other documents, e.g. if we want to get events which are part of exhibitions in `fetchExhibitions()`

    ```typescript
    const eventsFetchLinks = ['events.title', 'events.times'];
    ```

This PR attempts to avoid this confusion by settling on definition (2) as the canonical meaning of this variable.

This is part of a broader clean-up of fetchLinks, which is https://github.com/wellcomecollection/wellcomecollection.org/issues/8846 and required for https://github.com/wellcomecollection/wellcomecollection.org/issues/8830

### CODE REVIEW

File: content/webapp/services/prismic/fetch/exhibitions.ts [hunk 1]
```diff
@@ -146,7 +147,6 @@ export const fetchExhibitionRelatedContent = async (
     ...contributorsFields,
     ...eventSeriesFields,
     ...eventPoliciesFields,
-    ...contributorsFields,
```
Comment (rcantin-w) [inline content/webapp/services/prismic/fetch/exhibitions.ts:149]: Is this part of an extra cleanup of unused fields? (was it removed on purpose)

Comment (alexwlchan) [reply content/webapp/services/prismic/fetch/exhibitions.ts:149]: Not so much unused, as duplicated – look three lines up, it's used twice.

Comment (rcantin-w) [reply content/webapp/services/prismic/fetch/exhibitions.ts:149]: ahhh yes, getting cross eyed!

**Review by rcantin-w — APPROVED** [review]
Makes sense to me, and looks like it would've helped debug when we couldn't find where to pass in the series number, so 🙌 for making it clearer

**Pull request merged**